test/spec/functional/viewTransition.js

```diff
@@ -0,0 +1,59 @@
+var testSrcPath = '../../source',
+    spyOnBackbone = require('./backboneSpy');
+
+describe('A View that can transition in tracked views', function() {
+
+  var env, _, $, ParentView;
+
+  beforeEach(function(done) {
+    require('./clientEnv')().done(function(environment) {
+      env = environment;
```

Comment:
done. can't say i like how it ended up looking - most of what i was doing was in nested functions with different this contexts.